src/tools.json

```diff
@@ -65,7 +65,7 @@
             },
             "darwin-arm64": {
                 "url": "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.14.7/openshift-client-mac-arm64-4.14.7.tar.gz",
-                "sha256sum": "5f045d5aa7db7e3dd2b125c5f940112a541b77726ad99e9f6b00827f9dee1621",
+                "sha256sum": "d97b84603382e24c9927150b8bfe83be89b0e63f74a593d59022483a8d039600",
```

Comment:
The sha256sum of the downloaded `.tar.gz` does not match the expected sha256sum given on the OpenShift mirror website from which the tarball is downloaded.

Here is the sum reported by the OpenShift mirror website: https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.14.7/sha256sum.txt . It's the old one that Victor added (`5f045...`).

This might signal that the openshift executable for Apple Silicon is malicious.

Reply:
The checksum is taken from `https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.14.7/sha256sum.txt` file, from the following row:
```
5f045d5aa7db7e3dd2b125c5f940112a541b77726ad99e9f6b00827f9dee1621  openshift-client-mac-arm64-4.14.7.tar.gz
```
So, even if its value is  wrong, it's not our fault. 

Maybe it'll be better if we'll be really downloading and calculating the checksums when we check for the version updates?

PS: Or the value is really a correct one, but @rgrunber has some other data cached for the binary and as such performs the wrong calculation. Quite possible.